homework/Repository.js

```diff
@@ -12,9 +12,19 @@ class Repository {
    * Render the repository info to the DOM.
    * @param {HTMLElement} container The container element in which to render the repository.
    */
-  render(container) {
+
+  static assignLeftPanelValues(dataIndex) {
```

Comment:
Hi Utku - this method shouldn't necessarily be a static method. It should instead be a method on the class which can be accessed from an object. I think you tried doing the same thing we are doing in Utils.js but please know that it serves a different purpose there.

A rule of thumb while trying to identify when to make a method `static` or not is: ```does it make sense to call this method, even if no Obj has been constructed yet?```

If we go with the above rule of thumb, calling `assignLeftPanelValues` without a repository object doesn't make sense which is why we need it as a class method. 🙂

Reply:
Absolutely. I have replaced them with instance methods.